08/model.py

```diff
@@ -0,0 +1,82 @@
+import controller
+import interface
+import pandas
+import logger
+
+
+def menu_choice(data):
+    logger.logging.info("Вывод меню")
+    if data == '1':
+        view_file_csv()
+    elif data == '2':
+        add_row_in_file_csv()
+    elif data == '3':
+        change_row_in_file_csv()
+    elif data == '4':
+        delete_row_in_file_csv()
+    elif data == '5':
+        create_file_csv()
+    elif data == 'x':
+        exit()
```

Comment:
Можно было тут просто сделать `pass`. Но если хотите закрыть подобным образом, то лучше сделать вот так:
```suggestion
        sys.exit()
```

Reply:
Да да, все время забываю :)